component/authenticator/src/main/java/org/wso2/carbon/identity/policy/password/PasswordPolicyUtils.java

```diff
@@ -188,4 +206,230 @@ public static String convertWindowsFileTimeToUnixTime(String windowsFileTime) {
         long millisSinceEpoch = (fileTime / HUNDREDS_OF_NANOSECONDS) - WINDOWS_EPOCH_DIFF;
         return String.valueOf(millisSinceEpoch);
     }
+
+    public static boolean isPasswordExpiredForUser(String tenantDomain, double daysDifference,
+                                                   String passwordLastChangedTime, String tenantAwareUsername, UserStoreManager
+                                                           userStoreManager) throws AuthenticationFailedException,
+            org.wso2.carbon.user.core.UserStoreException {
+        try {
+            String userId = ((AbstractUserStoreManager) userStoreManager).getUserIDFromUserName(tenantAwareUsername);
+            List<PasswordExpiryRule> passwordExpiryRules =
+                    org.wso2.carbon.identity.password.expiry.util.PasswordPolicyUtils
```

Comment:
shall we move the class to the import section and simplify it as PasswordPolicyUtils..getPasswordExpiryRules(tenantDomain);

Reply:
We already have a PasswordPolicyUtils class in this repo itself. Hence the class is added like this